start.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Start script for company-accounts.web.ch.gov.uk
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/company-accounts.web.ch.gov.uk/env
+
+    PORT="${COMPANY_ACCOUNTS_WEB_PORT}"
+else
+    PORT="$1"
+    CONFIG_URL="$2"
+    ENVIRONMENT="$3"
+    APP_NAME="$4"
+
+    source /etc/profile
+
+    echo "Downloading environment from: ${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}"
+    wget -O "${APP_DIR}/private_env" "${CONFIG_URL}/${ENVIRONMENT}/private_env"
+    wget -O "${APP_DIR}/global_env" "${CONFIG_URL}/${ENVIRONMENT}/global_env"
+    wget -O "${APP_DIR}/app_env" "${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}/env"
+    source "${APP_DIR}/private_env"
+    source "${APP_DIR}/global_env"
+    source "${APP_DIR}/app_env"
+fi
+
+exec java ${JAVA_MEM_ARGS} -jar -Dserver.port="${PORT}" "${APP_DIR}/company-accounts.web.ch.gov.uk.jar"
```

Comment:
It may be worth adding the debug settings within the start script like we have done in the `company-accounts.api.ch.gov.uk`

Reply:
We're not going to add a debug command that is commented out. It deviates from our other `start` scripts, but if we did want to support this as standard then we'd add proper functionality with flags (or some other variation) as opposed to adding a commented out debug line.

Reply:
It is definitely worth adding or even possibly discussing. `./start.sh -d` to trigger the debug profile would be handy and additionally making it default to non-debug without it.

Reply:
I think this is worth a bigger discussion (tech meeting?), but I don't think this should block a PR going in to add *basic* start script functionality so that this service can be run (bear in mind there's no endpoints in the service at this point).